docs/changelog.rst

```diff
@@ -1 +1,7 @@
 .. include:: ../CHANGELOG.rst
+   :end-before: Changes for the upcoming release can be found
+
+.. towncrier-draft-entries:: |release|:sub:`/UNRELEASED DRAFT/`
```

Comment:
What exactly is the `sub` supposed to do? I would have guessed substitute the release but it looks like it uses a `<sub>` tag or something? Is that on purpose? If yes: why? :)

Reply:
Would you prefer something like this?
```suggestion
.. towncrier-draft-entries:: |release| | *UNRELEASED DRAFT*
```
Any other thoughts?

Reply:
maybe even remove `|release|` too? pip hasn't it either

Reply:
fee free to use their `Not yet released`

Reply:
Yeah, maybe even `Will be included in the upcoming release`?

Reply:
naw too long. what's good enough for pip, is good enough for us (in this case)

Reply:
I've gone for what pip has.